pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -79,86 +77,101 @@ In plain language, this code says:
-// Ex: IMG_DEST = "./assets/uswds/img"
-const IMG_DEST = "./path/to/images/destination";
-
-// Fonts destination
-// Where should we save USWDS fonts?
-// Ex: FONTS_DEST = "./assets/uswds/fonts"
-const FONTS_DEST = "./path/to/fonts/destination";
-
-// Javascript destination
-// Where should we save USWDS javascript?
-// Ex: JS_DEST = "./assets/uswds/js"
-const JS_DEST = "./path/to/js/destination";
-
-// Compiled CSS destination
-// Where should we save compiled USWDS CSS?
-// Ex: CSS_DEST = "./assets/uswds/css"
-const CSS_DEST = "./path/to/css/destination";
-
-// Site CSS destination
-// Like the _site/assets/css directory in Jekyll, if necessary.
-// If using, uncomment line 106
-// Ex: SITE_CSS_DEST = "./_site/assets/uswds/css"
-const SITE_CSS_DEST = "./path/to/site/css/destination";
-```
+/** 
+ * Path settings
+ * Set as many as you need
+ */ 
 
-Don’t include a trailing slash in any of these paths.
+uswds.paths.dist.css = './assets/css';
+uswds.paths.dist.sass = './sass';
 
-Save this file, and you’re ready to get started.
+/** 
+ * Exports
+ * Add as many as you need
+ */ 
 
-## Step 5: Initialize your project
-Initialize your project to copy all the necessary image, font, and Javascript assets from the source code. Initialization will also add a Sass entry point called `styles.scss` in the directory you set as `PROJECT_SASS_SRC` (as outlined in Step 3) for your project.
+exports.init = uswds.init;
+exports.compile = uswds.compile;
+  ```
+
+## Step 4: Create path settings and export compile functions
+Now, create your path settings using the following table:
+
+Setting | Default | Description
+--- | --- | ---
+`paths.src.uswds` | `"./node_modules/uswds/dist"` | Source location of the `uswds` package
+`paths.src.sass` | `"./node_modules/uswds/dist/scss"` | Source location of the USWDS Sass 
+`paths.src.theme` | `"./node_modules/uswds/dist/scss/theme"` | Source location of the USWDS theme files (Sass entry point and starter settings files)
+`paths.src.fonts` | `"./node_modules/uswds/dist/fonts"` | Source location of the USWDS fonts
+`paths.src.img` | `"./node_modules/uswds/dist/img"` | Source location of the USWDS images
+`paths.src.js` | `"./node_modules/uswds/dist/js"` | Source location of the USWDS compiled JavaScript files
+`paths.src.projectSass` | `"./sass"` | Source location of any exisiting project Sass files outside of `paths.dist.sass`. The `watch` script will watch this directory for changes.
+`paths.dist.sass` | `"./sass"` | Project destination for theme files (Sass entry point and settings)
+`paths.dist.img` | `"./assets/uswds/images"` | Project destination for images
+`paths.dist.fonts` | `"./assets/uswds/fonts"` | Project destination for fonts
+`paths.dist.js` | `"./assets/uswds/js"` | Project destination for compiled JavaScript
+`paths.dist.css` | `"./assets/uswds/css"` | Project destination for compiled CSS
+
+Note, the `src` settings are specific to the Design System; the `dist` settings are specific to your project.
```

Comment:
This big table breaks out of the content. Maybe this could link to `@uswds/compile` README or only include the most common paths.

Reply:
I'm hesitant to send people to a README and have users think they need to stay there.

Reply:
This table content does get cut off in narrow browser windows. Should we make the table scrollable?

Reply:
@amyleadem , If making it scrollable helps with accessibility, that sounds like a good resolution to me!